.storybook/themes/news.scss

```
@@ -835,6 +844,62 @@
 					padding: 0 var(--global-spacing-6) 0 0,
 					display: flex,
 				),
+				share-bar: (
+					border-radius: 0 5px 5px 0,
+					display: flex,
+					left: 0,
+					padding: 10px 0,
+					position: fixed,
+					top: 200px,
+					width: 44px,
+					components: (
+						button-medium: (
+							background: transparent,
+							border-width: 0,
+							margin: 5px,
+							padding: 0,
+						),
+						icon: (
+							height: var(--global-font-size-9),
+							width: var(--global-font-size-9),
+						),
+					),
+				),
+				share-bar-facebook: (
+					components: (
+						icon: (
+							fill: #4267b2,
+						),
+					),
+				),
```

If you apply these styles at the mobile breakpoint you can write this as 

```suggestion
				share-bar-facebook: (
					fill: #4267b2,
				),
```

---

Thanks, got it!